apps/frontend/src/pages/analytics/analytics.tsx

```diff
@@ -0,0 +1,42 @@
+import { DateInput, PageLayout } from "~/libs/components/components.js";
+import { useAppForm, useCallback } from "~/libs/hooks/hooks.js";
+
+import { THIRTY_DAYS } from "./libs/constants/constants.js";
+import { subtractDays } from "./libs/helpers/helpers.js";
+import styles from "./styles.module.css";
+
+const Analytics = (): JSX.Element => {
+	const { control, handleSubmit } = useAppForm({
+		defaultValues: {
+			dateRange: [subtractDays(new Date(), THIRTY_DAYS), new Date()],
```

Comment:
move todayDate to a separate constant
```suggestion
			dateRange: [subtractDays(todayDate, THIRTY_DAYS), todayDate],
```